.vscode/deepl-api-test.http

```diff
@@ -0,0 +1,7 @@
+POST https://api-free.deepl.com/v2/translate HTTP/1.0
+User-Agent: YourApp
+Accept: */*
+Content-Length: 54
+Content-Type: application/x-www-form-urlencoded
+
+text=%E6%8B%B7%E8%B4%9D%E8%AE%BE%E7%BD%AE&source_lang=zh&target_lang=en&auth_key={{$dotenv [%]DEEPL_FREE_API_KEY}}
```

Comment:
(looks okay, but I didn't get it to work in the few minutes I tried. I had to copy the key in :)

Reply:
What is this file for?

Reply:
"humao.rest-client" VSCode extentension (https://github.com/Huachao/vscode-restclient)

Reply:
(perhaps it shouldn't be there...)

Reply:
(I wanted to comment, not review, I'll try to get rid of it)